packages/connect-examples/new-example/app/components/common/ParameterInput.tsx

```
@@ -0,0 +1,511 @@
+                  param.name === 'bundle',
+              })),
+            }
+          : null,
+      });
+
+      if (preset && preset.parameters) {
+        console.log(
+          '[ParameterInput] 预设参数:',
+          preset.parameters.map(param => ({
+            name: param.name,
+            type: param.type,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)